src/python/pants/backend/codegen/protobuf/python/python_protobuf_subsystem.py

```diff
@@ -39,6 +39,26 @@ class PythonProtobufSubsystem(Subsystem):
         """
     )
 
+    grpcio_plugin = BoolOption(
```

Comment:
Does it ever make sense to use both plugins? if not, maybe this should be an enum with none/grpcio/grpclib options?

Reply:
While probably uncommon, I think it might happen, especially given that these options are global to the whole project. I can see some monorepo to have some project using grpclib and other using grpcio due to migration or other internal requirements. There would be no escape hatch for these cases if we have an enum. 

Having the enum on `protobuf_source` would have work since it would have been possible to use both on different proto (Or even have two `protobuf_source` on the same proto file with each needed plugin).

Another alternative would have been to make this enum option a list, so something like:

```toml
[python-protobuf]
mypy_protobuf = true
protoc_plugins = ["grpclib"]
protoc_plugins = ["grpclib", "grpcio"] # if you really want both
```

I just feel it adds some inconsistency in the ways to enable plugins. Some plugin have their own bool (`mypy_protobuf`) and other are passed as a value (`protoc_plugins`).

To be honest, if I were to design it from scratch I would have liked to do something along:

```
[protobuf]
protoc_plugins = [
  "pants.backend.codegen.protobuf.python.grpc",
  "pants.backend.codegen.protobuf.python.mypy",
  "pants.backend.codegen.protobuf.java",
]
```

which would have allowed to implement the `grpclib` (and any other third-party plugin support) as a simple pants plugin, but I'm not familiar enough with pants if it's even possible to do this, and how to do it elegantly.

Reply:
Makes sense, we can leave as-is for now.